modules/nf-core/umicollapse/tests/main.nf.test

```diff
@@ -13,57 +13,16 @@ nextflow_process {
     tag "bwa/mem"
```

Comment:
We can remove the tags then - can't comment on the lines above.

Reply:
Done. Thanks a lot for the eagle eye!

Reply:
That'd be the point of reviewing ;)